### PR TITLE
Change Testerina configuration details

### PR DESCRIPTION
**Pull request by Ibaqu**
## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/testing-ballerina-code/writing-tests.md [hunk 1]
```diff
@@ -639,8 +639,11 @@ function afterFunc() {
 
 ## Test Configurations
 
-Configurations for testing can be provided using the Config API. For information on using the config library, see
- [Config Module](/swan-lake/learn/api-docs/ballerina/config/index.html).
+Configurations for testing can be provided using Configurable Variables. The values for configurable
+variables can be provided in a file named `configuration.toml` located in the tests directory.
```
Comment (HindujaB) [inline swan-lake/learn/testing-ballerina-code/writing-tests.md:643]:
```suggestion
variables can be provided in a file named `Config.toml` located in the tests directory.
```

File: swan-lake/learn/testing-ballerina-code/writing-tests.md [hunk 2]
```diff
@@ -639,8 +639,11 @@ function afterFunc() {
 
 ## Test Configurations
 
-Configurations for testing can be provided using the Config API. For information on using the config library, see
- [Config Module](/swan-lake/learn/api-docs/ballerina/config/index.html).
+Configurations for testing can be provided using Configurable Variables. The values for configurable
```
Comment (HindujaB) [inline swan-lake/learn/testing-ballerina-code/writing-tests.md:642]:
```suggestion
Configurations for testing can be provided using configurable variables. The values for configurable
```

File: swan-lake/learn/testing-ballerina-code/writing-tests.md [hunk 3]
```diff
@@ -639,8 +639,11 @@ function afterFunc() {
...
+variables can be provided in a file named `configuration.toml` located in the tests directory.
+
+For information on using Configurable variables, see
```
Comment (HindujaB) [inline swan-lake/learn/testing-ballerina-code/writing-tests.md:645]:
```suggestion
For information on using configurable variables, see
```

File: swan-lake/learn/testing-ballerina-code/writing-tests.md (hunk 1 above)
Comment (HindujaB) [reply swan-lake/learn/testing-ballerina-code/writing-tests.md]: The file name starts with uppercase

File: swan-lake/learn/testing-ballerina-code/writing-tests.md [hunk 4]
```diff
@@ -639,8 +639,11 @@ function afterFunc() {
 
 ## Test Configurations
 
-Configurations for testing can be provided using the Config API. For information on using the config library, see
- [Config Module](/swan-lake/learn/api-docs/ballerina/config/index.html).
+Configurations for testing can be provided using configurable Variables. The values for configurable
```
Comment (HindujaB) [inline swan-lake/learn/testing-ballerina-code/writing-tests.md:642]:
```suggestion
Configurations for testing can be provided using configurable variables. The values for configurable
```

Comment (praneesha) [inline swan-lake/learn/testing-ballerina-code/writing-tests.md:642]:
```suggestion
Configurations for testing can be provided using configurable variables. The values for configurable
```

File: swan-lake/learn/testing-ballerina-code/writing-tests.md (hunk 1 above)
Comment (Ibaqu) [reply swan-lake/learn/testing-ballerina-code/writing-tests.md]: Changed to uppdercase Config.toml

File: swan-lake/learn/testing-ballerina-code/writing-tests.md (hunk 4 above)
Comment (Ibaqu) [reply swan-lake/learn/testing-ballerina-code/writing-tests.md]: Changed to lowercase

**Review by praneesha — APPROVED** [review]

**Pull request merged**